docs/Tutorial_Contrast_Profile2.ipynb

```diff
@@ -0,0 +1,498 @@
+{
```

Comment:
<p>Maybe remove the line <code>Define index of desired instances on the purpose</code>?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/800/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>